libcxx/include/__format/container_adaptor.h

```diff
@@ -18,11 +18,11 @@
 #include <__format/concepts.h>
 #include <__format/formatter.h>
 #include <__format/range_default_formatter.h>
+#include <__fwd/queue.h>
+#include <__fwd/stack.h>
```

Comment:
Why do we need place these concrete `formatter` specializations in a concentrated header?
IIRC these specializations should be placed in `<stack>` and `<queue>`, see cplusplus/draft#6635. (The change was not so editorial to me, but Daniel told me it was.)

Reply:
IDK, but this seems not really related to the change made here. FWIW the change doesn't look editorial to me either.